### PR TITLE
release: v0.4.0

### PR DESCRIPTION
**Pull request by jirikuncar**
Signed-off-by: Jiri Kuncar jiri.kuncar@cern.ch

### CODE REVIEW

**Comment by tiborsimko** [comment]
FWIW we may want to hold on the release a little bit... I have another mega-PR coming to fix `pep257` everywhere, including in `flask-cli`.  Wanted to have the `sphinx 1.4` fixes merged first, so that the builds would pass...

https://github.com/inveniosoftware/flask-breadcrumbs/pull/26#discussion_r67866049

**Comment by lnielsen** [comment]
FYI Once we release this one, I potentially can break quite a lot of code on @inveniosoftware so perhaps worthwhile just figuring out what to do before releasing. E.g. will it be possible to run existing Invenio modules on both v0.3.0 + 0.4.0? If not, then we have to upgrade all Invenio modules to use v0.4.0 in one go, as we otherwise create a situation where some Invenio modules only works with 0.4.0 and others only works with 0.3.0 (while being installed in the same env 💣).

I'm not objecting to the release. Just advocating we do a bit of up-front checking of what problems we will have.

File: CHANGES [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+- Updates help to `> set FLASK_APP=hello.py`.
+
+- Adds plugin support to the CLI.
+
+- Updates docs to the new CLI patterns.
+
+- Removes Python 2.6 and adds Python 3.5 to Tox configuration to match
+  Travis config.
+
+- Implementeds simplified CLI interface.
```
Comment (tiborsimko) [inline CHANGES]: `s/Implementeds/Implements`

**Comment by jirikuncar** [comment]
@lnielsen all Invenio-\* packages are tested with Flask-CLI 0.2.1, latest release, and GitHub version. Hence, I'm not expecting any problem.

**Pull request merged**